### ISSUE TITLE
Taylor + Lightweight ORM

### ISSUE THREAD

**Issue by levin-ilya** [izqui/Taylor#20]
Anyone interested in starting a lightweight ORM to work with Taylor?

Curious to know people thoughts on this...

Thanks

**Comment by Danappelxx** [comment]
I've been slowly working on a modular web-framework built around Taylor called [Mast](https://github.com/Mast-Swift/). It's not anywhere near alpha and right now the Readme doesn't even reflect what I want it to be, but eventually it will be something great. Personally (I'm biased) I think that no library should be all-inclusive, so if you make an ORM for Taylor I encourage you to create it as a completely separate (but combosable) library, or as a module for Mast (which would be absolutely amazing!).

tl;dr: Do it modularly or don't do it at all 😛

**Comment by izqui** [comment]
+1

**Comment by levin-ilya** [comment]
Thanks, I will check out Mast! I'm in the middle of an iPhone project now, but once i'm done i'll have some time to help out. So if someone else wants to start it, this is fine as well.

Very excited about all these projects!
